int64 servo_x
int64 servo_y